### PR TITLE
docs(kubernetes) update for Kong v0.11

### PR DESCRIPTION
**Pull request by shashiranjan84**

### CODE REVIEW

File: app/install/kubernetes.md [hunk 1]
```diff
@@ -46,18 +52,20 @@ Kong can easily be provisioned to Kubernetes cluster using the following steps:
 
     By now, you have provisioned a Kubernetes managed cluster.
 
+
 3. **Deploy a Kong supported database**
   
-    Before deploying Kong, you need to provision a Cassandra or PostgreSQL pod.
+    Before deploying Kong, you need to provision a Cassandra or PostgreSQL pods.
```
Comment (p0pr0ck5) [inline app/install/kubernetes.md]: why the change from "pod" to "pods"? grammar issue here with "a ... pod*s*"

File: app/install/kubernetes.md [hunk 2]
```diff
@@ -20,24 +21,29 @@ Kong can easily be provisioned to Kubernetes cluster using the following steps:
...
+    please use the [tag 1.0.0][gh_tag]
+
 2.  **Deploy a GKE cluster**
     
-    You need [gcloud]({{ page.links.gcloud }}) and
-    [kubectl]({{ page.links.kubectl }}) command-line tools installed and set up
-    to run deployment commands. Also make sure your Google Cloud account has at
-    least two `STATIC_ADDRESSES` available. 
+    You need [gcloud][gcloud] and [kubectl][kubectl] command-line tools
+    installed and set upto run deployment commands. Also make sure your Google
```
Comment (p0pr0ck5) [inline app/install/kubernetes.md]: space between up and to?

Comment (shashiranjan84) [inline app/install/kubernetes.md]: Updated, thanks @p0pr0ck5

File: app/install/kubernetes.md [hunk 3]
```diff
@@ -66,37 +74,49 @@ Kong can easily be provisioned to Kubernetes cluster using the following steps:
     $ kubectl create -f postgres.yaml
     ```
 
-4. **Deploy Kong**
+4. **Prepare database**
+
+    Using the `kong_migration_<postgres|cassandra>.yaml` file from this repo,
+    run the migration job, jump to step 5 if Kong backing databse is up–to–date:
```
Comment (salazar) [inline app/install/kubernetes.md]: "and jump to step 5 if Kong database is up-to-date"; also, is "up-to-date" the best term here? Maybe "ready" instead?

File: app/install/kubernetes.md [hunk 4]
```diff
@@ -66,37 +74,49 @@ Kong can easily be provisioned to Kubernetes cluster using the following steps:
...
+    $ kubectl delete -f kong_migration_<postgres|cassandra>.yaml
+    ```
+
+5. **Deploy Kong**
 
-    Using the `kong_<postgres|cassandra>.yaml` file from the
-    kong-dist-kubernetes repo, deploy a Kong `Service` and a `Deployment` to the
-    cluster created in the last step:
+    Using the `kong_<postgres|cassandra>.yaml` file from this
+    repo, deploy Kong admin and proxy services and a `Deployment` controller to
```
Comment (salazar) [inline app/install/kubernetes.md]: "deploy Kong admin, proxy services, and a (...)"; what do we mean by "deploy Kong admin"? Could it be "deploy Kong"?

Comment (shashiranjan84) [inline app/install/kubernetes.md]: Those are separate services and deployed as a separate resources.
